script/foreman-debug

```diff
@@ -22,7 +22,7 @@ OPTIONS:
   -d DIR  Directory to place the tarball in (default /tmp/foreman-XYZ)
   -g      Skip generic info (CPU, memory, firewall etc.)
   -a      Do not generate a tarball from the resulting directory
-  -m NUM  Maximum lines to keep for each file (default 5000), zero means no limit
+  -m NUM  Maximum size in kB for each file type (default 5000 kB), zero means no limit
```

Comment:
Oh I noticed this is kB, therefore 5 MB per log file or directory provided? I think we can go higher here. Ten to twenty would be reasonable I think.